store/timerlayer/timerlayer.go

```diff
@@ -4821,6 +4821,22 @@ func (s *TimerLayerOAuthStore) RemoveAuthData(code string) error {
 	return err
 }
 
+func (s *TimerLayerOAuthStore) RemoveMultipleAccessData(tokens []string) error {
+	start := time.Now()
+
+	err := s.OAuthStore.RemoveMultipleAccessData(tokens)
+
+	elapsed := float64(time.Since(start)) / float64(time.Second)
+	if s.Root.Metrics != nil {
```

Comment:
This code and the one starting at 7052 is almost the same except one parameter, wouldn't it be better to create a private method to DRY?

Reply:
Also 7132

Reply:
This code is generated